distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/SystemJvmOptions.java

```diff
@@ -82,6 +79,16 @@ static List<String> systemJvmOptions(Settings nodeSettings, final Map<String, St
         ).filter(e -> e.isEmpty() == false).collect(Collectors.toList());
     }
 
+    @UpdateForV9    // only use CLDR in v9+
+    private static String getLocaleProviders() {
+        /*
+         * Specify SPI to load IsoCalendarDataProvider (see #48209), specifying the first day of week as Monday.
```

Comment:
As a followup, do we still need this calendar provider once we switch to CLDR? Given one reason we are switching to CLDR (instead of reimplementing COMPAT) to follow standards, we should revisit whether such an override is appropriate.